web/src/theme/NavbarItem/useMediaQueryAuth0.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+export function useMediaQueryAuth0 () {
+  const { loginWithPopup, loginWithRedirect, ...others } = useAuth0();
+  const matches = useMediaQuery('(min-width:600px)');
```

Comment:
useMediaQuery(theme => theme.breakpoints.down('md'))

Reply:
The problem is with mobile but not screen size, maybe we need to check user agent?

Reply:
- `/subscriptions` has no `theme`.
- Check screen size is more easier than user agent. We need more investigations about detecting `user-agent`.